HomeWorkTask20_Library/src/com/ilyafrolov/lesson20/Library.java

```diff
@@ -0,0 +1,68 @@
+package com.ilyafrolov.lesson20;
+
+import java.util.*;
+
+public class Library {
+
+    List<Book> books;
+
+    public Library() {
+        books = new LinkedList();
+    }
+
+    public void addBook(Book book) {
+        boolean isUnique = true;
+        if (books.size() == 0) {
+            books.add(book);
+        } else {
+            for (int i = 0; i < books.size(); i++) {
+                if (books.get(i).equals(book)) {
+                    isUnique = false;
+                }
+            }
+            if (isUnique) {
+                books.add(book);
+            }
+        }
+    }
+
+    public void printListOfBooks() {
+        Iterator iterator = books.iterator();
+        while (iterator.hasNext()) {
+            System.out.println(iterator.next());
+        }
+    }
+
+    public String toString() {
+        return books.toString();
+    }
+
+    public void sortByTitle() {
+        Collections.sort(books);
```

Comment:
It's not obvious that sorting will be made by title. It's better to create here comparator (or even better to create comparator at the class level (to be created just once) and use it here - similar with sorting by date but with comparator creation on class level

Reply:
Done.